libsnark/zk_proof_systems/plonk/prover.tcc

```diff
@@ -0,0 +1,1186 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#ifndef LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_PROVER_TCC_
+#define LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_PROVER_TCC_
+
+/// Implementation of Prover interfaces for a ppzkSNARK for Plonk. See
+/// prover.hpp .
+
+namespace libsnark
+{
+
+/// Prover Round 0 initialization
+///
+/// Initialization
+///
+/// INPUT
+/// \param[in] srs: structured reference string containing also
+///            circuit-specific information
+///
+/// OUTPUT
+/// \param[out] W_polys: Lagrange interpolation of the witness values
```

Comment:
Doesn't seem to appera in the output.

Reply:
Indeed. W_polys is part of the round 1 output. Fixed. Thanks!